pkg/cmd/pipeline/translate_test.go

```diff
@@ -110,8 +110,49 @@ func Test_AddDevAnnotations(t *testing.T) {
 	assert.Equal(t,
 		d.Annotations,
 		map[string]string{
-			model.OktetoRepositoryAnnotation: "git-repo",
+			model.OktetoRepositoryAnnotation: "file://git-repo",
```

Comment:
Following my assumption that changing the value of this annotation does not affect other existing logic, I have modified this test instead of adding more logic to the introduced function just to pass the test